src/chapters/03/main.tex

```diff
@@ -105,8 +105,8 @@ \section{Theory}\label{sec:discrete_event_simulation_theory}
 They can however generate pseudorandom numbers: sequences of numbers that look
 like random numbers, but are entirely determined from the
 previous numbers in the sequence\footnote{An early
-discussion of pseudo random numbers is~\cite{von195113} where the author
-claimed: ``Anyone who considers arithmetical methods of producing random digits
+discussion of pseudorandom numbers is~\cite{von195113} where the author
+claimed: ``Anyone who considers arithmetical methods of producing random digits % TODO I do not understand this quite and don't think it's relevant.
 is, of course, in a state of sin.'' A number of different pseudo random
```

Comment:
I thought it was a neat/cute little quote: it's just saying that "you're wrong to even think about producing randomness with mathematical methods but ... [here is how to do it]". I'm not attached to it though, if you don't like it we can just remove it

```suggestion
discussion of pseudorandom numbers is~\cite{von195113}. A number of different pseudo random
```